packages/angular/generators.json

```diff
@@ -86,7 +86,7 @@
       "schema": "./src/generators/move/schema.json",
       "aliases": ["mv"],
       "description": "Moves an Angular application or library to another folder within the workspace and updates the project configuration.",
-      "x-deprecated": "Use the `@nx/workspace:move` generator instead. This generator will be removed in Nx v19."
+      "x-deprecated": "Use the `@nx/workspace:move` generator instead. This generator will be removed in Nx v22."
```

Comment:
Is this bumped simply because we missed it?

Reply:
We didn't have a proper deprecation notice in the programmatic API, so I'm adding it to this PR and extending the notice until the next major. It's a generator we haven't touched in a while, so it's low maintenance at this point. We can live with it for another major.

Reply:
Sounds good!